library/core/tests/slice.rs

```diff
@@ -2504,3 +2504,17 @@ fn test_slice_from_ptr_range() {
         assert_eq!(slice::from_ptr_range(range), &arr);
     }
 }
+
+#[test]
```

Comment:
Aha!  I figured out what's wrong -- the PR build is failing because it's `check`ing these tests against `stage0`, and you're calling a method that depends on the new `impl` resolution that's not in `stage0`, so I think you need to `cfg` them out of the bootstrap checks, like
```suggestion
#[cfg(not(bootstrap))]
#[test]
```